### ISSUE TITLE
Import into Angular Project

### ISSUE THREAD

**Issue by albertoherraiz** [Ovilia/ECharts-Theme-Builder#18]
Hi! How can you import it to an angular project? 

Regards

**Comment by Ovilia** [comment]
@albertoherraiz Do you mean the theme builder or ECharts project?

**Comment by albertoherraiz** [comment]
Hi @Ovilia ! I mean the theme builder. Imagine that I make a theme with https://www.echartsjs.com/theme-builder/ and export it. 
Then this theme I want it to import it into an angular project. 

Regards

**Comment by Ovilia** [comment]
You can use the exported theme in an angular project right now.

**Comment by albertoherraiz** [comment]
Yes @Ovilia , I export it in two ways (json file + js file). But my question is: How can I add this files into an angular project and set it up? 

Regards

**Comment by Ovilia** [comment]
Hi @albertoherraiz Please try searching Angular import js or json.
